### PR TITLE
libshout, add extra options

### PR DESCRIPTION
**Pull request by Begasus**
Checked a clean build for mpd_x86 with libshout enabled, that worked, could use some testing to make sure :)

### CODE REVIEW

File: media-libs/libshout/libshout-2.4.1.recipe [hunk 1]
```diff
@@ -10,9 +10,14 @@ COPYRIGHT="2002-2012 the Icecast team
 	2014 Brendan Cully
 	1995-1997 by Sam Rushing"
 LICENSE="GNU LGPL v2.1"
-REVISION="3"
+REVISION="4"
 SOURCE_URI="http://downloads.xiph.org/releases/libshout/libshout-$portVersion.tar.gz"
 CHECKSUM_SHA256="f3acb8dec26f2dbf6df778888e0e429a4ce9378a9d461b02a7ccbf2991bbf24d"
+PATCHES="libshout-$portVersion.patchset"
+if [ "$effectiveTargetArchitecture" = x86_gcc2 ]; then
+	PATCHES="$PATCHES
```
Comment (korli) [inline media-libs/libshout/libshout-2.4.1.recipe]: you could use: PATCHES+="libshout-$portVersion-gcc2.patchset"

File: media-libs/libshout/libshout-2.4.1.recipe [hunk 2]
```diff
@@ -23,7 +28,11 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
 	lib:libvorbis$secondaryArchSuffix
+	lib:libtheora$secondaryArchSuffix
```
Comment (korli) [inline media-libs/libshout/libshout-2.4.1.recipe]: lib:libvorbis comes after lib:libtheora, etc..

File: media-libs/libshout/libshout-2.4.1.recipe [hunk 3]
```diff
@@ -49,8 +66,13 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
+defineDebugInfoPackage libshout$secondaryArchSuffix \
+	$libDir/libshout.so.3.2.0
+
 BUILD()
 {
+	export LIBS="-lnetwork"
+	echo "AC_CONFIG_MACRO_DIRS([m4])" >> configure.ac
```
Comment (korli) [inline media-libs/libshout/libshout-2.4.1.recipe]: For each build, this would append this line.
Maybe move this to PATCH().

Comment (Begasus) [reply media-libs/libshout/libshout-2.4.1.recipe]: Will check later, thanks! :+1:

Comment (Begasus) [reply media-libs/libshout/libshout-2.4.1.recipe]: Moved it to PATCH()

**Pull request merged**